Makefile

```diff
@@ -89,6 +91,7 @@ test-functional:
 	@echo no functional tests
 
 test-style: check-docker
+	${DEV_ENV_CMD} shellcheck $(SHELL_SCRIPTS)
```

Comment:
@mboersma what would you think about moving the shellcheck under the `style-check` target that this tagret already runs within a container (see next line)?  This way we're slightly more efficient by only bootstrapping the container once and executing _all_ style checks in that one container.